### DISCUSSION TITLE
Strange behavior with auto-dark-mode

### DISCUSSION THREAD

**Discussion by dpo** [LunarVim/LunarVim#3981] [General]
Hi. I'm on macOS and I use `f-person/auto-dark-mode.nvim` to automatically switch to a light or dark theme when the OS does.
On a fresh LunarVim release-1.2 install, I noticed that with the following configuration:
```lua
lvim.colorscheme = "tokyonight-day"

lvim.plugins = {
  { "f-person/auto-dark-mode.nvim" },
}

local auto_dark_mode = require('auto-dark-mode')

auto_dark_mode.setup({
  update_interval = 1000,
  set_dark_mode = function()
    vim.api.nvim_set_option('background', 'dark')
    vim.cmd('colorscheme tokyonight-night')
  end,
  set_light_mode = function()
    vim.api.nvim_set_option('background', 'light')
    vim.cmd('colorscheme tokyonight-day')
  end,
})
auto_dark_mode.init()
```
all seems to work well until I make a change (any change, unrelated to `auto-dark-mode`) to `config.lua` and *save*. At the point where I save the config file, the theme resets to the default at the top of the snippet above (in my example, `tokyonight-day`).

I'm not sure whether that behavior is due to a bug in LunarVim or in auto-dark-mode. I wanted to see if others can reproduce it.

**Comment by LostInTheLogs** [comment]
when you save your config :LvimReload is run, unless you disable reload config on save and it will change the theme to lvim.colorscheme

**Comment by dpo** [comment]
But then, it's ignoring the auto-dark-mode part of my config.
